temporian/implementation/numpy/operators/tests/calendar_day_of_month_test.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import absltest
+import numpy as np
+
+from temporian.core.data.event import Event
+from temporian.core.data.event import Feature
+from temporian.core.data.sampling import Sampling
+from temporian.core.operators.calendar_day_of_month import (
+    CalendarDayOfMonthOperator,
+)
+from temporian.implementation.numpy.data.event import NumpyEvent
+from temporian.implementation.numpy.data.event import NumpyFeature
+from temporian.implementation.numpy.data.sampling import NumpySampling
+from temporian.implementation.numpy.operators.calendar_day_of_month import (
+    CalendarDayOfMonthNumpyImplementation,
+)
+
+
+class CalendarDayNumpyImplementationTest(absltest.TestCase):
```

Comment:
`CalendarDayOfMonthNumpyImplementationTest` too long but it is what it is 😆

Reply:
missed it, changing it, nice catch